tests/utils/datatype_tests.py

```diff
@@ -55,6 +56,50 @@ def test_tile_transform(self):
         with self.assertRaises(ValueError):
             boolean.transform_value_for_tile(None)
 
+class GeoJsonDataTypeTest(ArchesTestCase):
+
+    def test_validate_reduce_byte_size(self):
+        f = open("tests/fixtures/problematic_excessive_vertices.geojson")
+        geom = json.load(f)
+        geom_datatype = DataTypeFactory().get_instance("geojson-feature-collection")
+        errors = geom_datatype.validate(geom)
+        self.assertEqual(len(errors), 0)        
+
+
+    def test_validate_bbox(self):
+        settings.DATA_VALIDATION_BBOX = [(
```

Comment:
This has the potential to leave side effects on the next test. The doc'd way to do this in django is with `@override_settings`, but b/c arches subclasses the django settings, you also have to wrap the action your test takes with `@sync_overridden_test_settings_to_arches`, or possibly use it as a second decorator, although I haven't explored that.

There's an example in `test_arches_application_extension_implicit`.